app/services/info_store.rb

```diff
@@ -152,15 +152,15 @@ def median_note_length
     query = <<-SQL
       SELECT
         percentile_disc(0.5)
-        WITHIN group (order by note_size)
+        WITHIN group (order by note_size.column_size)
         FROM
           (SELECT
-             pg_column_size(annotation) 
+             pg_column_size(annotation) as column_size
```

Comment:
Guessing this (good) change is from stricter parsing in the upgraded postgres version